core/src/test/java/org/web3j/protocol/core/RequestTest.java

```diff
@@ -73,6 +73,13 @@ public void testNetPeerCount() throws Exception {
         verifyResult("{\"jsonrpc\":\"2.0\",\"method\":\"net_peerCount\",\"params\":[],\"id\":1}");
     }
 
+    @Test
+    public void testAdminNodeInfo() throws Exception {
+        web3j.adminNodeInfo().send();
+
+        verifyResult("{\"jsonrpc\":\"2.0\",\"method\":\"admin_NodeInfo\",\"params\":[],\"id\":1}");
```

Comment:
is it supposed to be `admin_nodeInfo`? - it is probably why the test is failing :)